test_util/test_upgrade_vpc.py

```diff
@@ -307,7 +307,9 @@ def setup_cluster_workload(self, dcos_api: DcosApiSession, healthcheck_app: dict
 
             dcos_api.marathon.deploy_app(viplisten_app)
             dcos_api.marathon.ensure_deployments_complete()
-            dcos_api.marathon.deploy_app(viptalk_app)
+            # viptalk app depends on VIP from viplisten app, which may still fail
```

Comment:
Huh? Those apps are already tested by that method if I am not mistaken. Look again. dns_app is just the only one passed as an arg. Everything else doesnt need to have the app definition passed because the test harness gets the original task id's at the beginning and checks them at the end for all 4 apps. With the dns_app there are some special value that shouldn't be redeclared so the definition is passed